packages/react-qr/src/ScanAddress.tsx

```diff
@@ -43,6 +43,8 @@ function ScanAddress ({ className, isEthereum, onError, onScan, size, style }: P
 
           onScan({ content, genesisHash, isAddress, name: name?.length ? name.join(':') : undefined });
         } catch (error) {
+          onError && onError(error);
```

Comment:
```suggestion
          onError && onError(error as Error);
```

We are sure here that we indeed have one, however TS have it as `unknown` (I believe)